CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - Log a warning when dropping envelopes due to rate-limiting (#4463)
 - Expose `SentrySessionReplayIntegration-Hybrid.h` as `private` (#4486)
 - Add `maskedViewClasses` and `unmaskedViewClasses` to SentryReplayOptions init via dict (#4492)
+- Add `quality` to SentryReplayOptions init via dict (#4495)
```

Comment:
Why does this need a changelog entry?

Reply:
It's internal, only for hybrids, so yes, we could omit it. But it's a bit easier to see when this was added for the hybrids SDKs dev.